resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/DriverServiceFeatureStep.scala

```diff
@@ -39,16 +40,17 @@ private[spark] class DriverServiceFeatureStep(
       "managed via a Kubernetes service.")
 
   private val preferredServiceName = s"${kubernetesConf.resourceNamePrefix}$DRIVER_SVC_POSTFIX"
-  private val resolvedServiceName = if (preferredServiceName.length <= MAX_SERVICE_NAME_LENGTH) {
-    preferredServiceName
-  } else {
-    val randomServiceId = KubernetesUtils.uniqueID(clock = clock)
-    val shorterServiceName = s"spark-$randomServiceId$DRIVER_SVC_POSTFIX"
-    logWarning(s"Driver's hostname would preferably be $preferredServiceName, but this is " +
-      s"too long (must be <= $MAX_SERVICE_NAME_LENGTH characters). Falling back to use " +
-      s"$shorterServiceName as the driver service's name.")
-    shorterServiceName
-  }
+  private val resolvedServiceName =
+    if (preferredServiceName.length <= KUBERNETES_LABEL_MAX_LENGTH) {
```

Comment:
Well, shall we minimize the PR by not touching this style, @yaooqinn ?

Reply:
In this block, technically, what we need is to replace with `MAX_SERVICE_NAME_LENGTH ` with `KUBERNETES_LABEL_MAX_LENGTH`? at two lines, isn't it?

Reply:
val MAX_SERVICE_NAME_LENGTH  = KUBERNETES_LABEL_MAX_LENGTH to keep these lines unchanged